src/pages/index.tsx

```diff
@@ -1,109 +1,15 @@
-import HeaderLayout from "@/layouts/HeaderLayout";
 import type { NextPage } from "next";
-import {
-  Flex,
-  Box,
-  Heading,
-  Container,
-  Text,
-  Button,
-  Stack,
-  Icon,
-  useColorModeValue,
-  createIcon,
-  Link,
-} from "@chakra-ui/react";
-import NoFoundIcon from "@/assets/404.svg";
+import HeaderLayout from "../layouts/HeaderLayout";
+import { Heading } from "@chakra-ui/react";
 
 const Home: NextPage = () => {
   return (
-    <HeaderLayout title="Homepage">
-      <Flex
-        direction="column"
-        alignItems="center"
-        justifyContent="center"
-        minHeight="85vh"
-        gap={4}
-        mb={8}
-        w="full"
-      >
-        <NoFoundIcon />
-        <Container maxW={"3xl"}>
-          <Stack
-            as={Box}
-            textAlign={"center"}
-            spacing={{ base: 8, md: 14 }}
-            py={{ base: 20, md: 36 }}
-          >
-            <Heading
-              fontWeight={600}
-              fontSize={{ base: "2xl", sm: "4xl", md: "6xl" }}
-              lineHeight={"110%"}
-            >
-              Master Java in 7 days <br />
-              <Text as={"span"} color={"green.400"}>
-                for Beginer
-              </Text>
-            </Heading>
-            <Text color={"gray.500"}>
-              Master Java Coding! This course requires no previous programming or Java experience.
-            </Text>
-            <Stack
-              direction={"column"}
-              spacing={3}
-              align={"center"}
-              alignSelf={"center"}
-              position={"relative"}
-            >
-              <Link
-                href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
-                isExternal
-                style={{ textDecoration: "none" }}
-              >
-                <Button variant={"exampleBtn"}>Get Started</Button>
-              </Link>
-              <Button variant={"link"} colorScheme={"blue"} size={"sm"}>
-                Learn more
-              </Button>
-              <Box>
-                <Icon
-                  as={Arrow}
-                  color={useColorModeValue("gray.800", "gray.300")}
-                  w={71}
-                  position={"absolute"}
-                  right={-71}
-                  top={"10px"}
-                />
-                <Text
-                  fontSize={"lg"}
-                  fontFamily={"Caveat"}
-                  position={"absolute"}
-                  right={"-150px"}
-                  top={"-15px"}
-                  transform={"rotate(10deg)"}
-                >
-                  Starting at $0.01/mo
-                </Text>
-              </Box>
-            </Stack>
-          </Stack>
-        </Container>
-      </Flex>
+    <HeaderLayout>
+      <Heading as="h1" size="xl" paddingLeft="326px">
```

Comment:
paddingLeft="326px"?
I know it is just a placeholder but it doesn't make any sense to make it padding left at 326px.

Reply:
I will fix that